docs/framework/tutorial/install.md

```diff
@@ -25,3 +59,32 @@ export PATH="$HOME/.local/bin:$PATH"
 ```
 
 [nodejs-download]: https://nodejs.org/en/download/
+
+## Installing Hatch Previewer ##
+
+You'll also need a tool called Hatch Previewer.
+It's available on Flathub, so if you're following this tutorial on Endless OS, you can install it from the App Center, or use the command line:
+
+```bash
+flatpak install flathub com.endlessm.HatchPreviewer
+```
+
+If you're on a different OS, you can follow [Flathub's instructions][flathub-install] to add it as a software source, or install Hatch Previewer via NPM.
+If you choose the latter, the way to run it will be slightly different.
+
+[flathub-install]: https://flathub.org/
+
+## Installing the Source Sans Pro fonts ##
+
+We'll be using the Source Sans Pro font in our app.
+You can find the latest release on its [GitHub page][source-sans-pro-releases].
+Download the `.otf` versions of the fonts.
+
+We'll build these fonts into our apps, but in order for Hatch Previewer to find them during development, you should also copy them into your local fonts directory:
+
+```bash
+mkdir -p ~/.fonts
```

Comment:
I think it's `~/.local/share/fonts` on modern distros (`~/.fonts` is still supported but legacy/deprecated at this point)

Reply:
Hatch previewer doesn't find them there. (I think it's probably due to Electron using outdated versions of the stack.)

Reply:
That's quite odd. It does not find them when running as a flatpak, or otherwise?

Reply:
Flatpak.